plugins/subnet/unit_test/test_calculate.py

```diff
@@ -0,0 +1,159 @@
+from unittest import TestCase
+from komand_subnet.actions.calculate import Calculate
+from komand_subnet.actions.calculate.schema import Input, Output
+from insightconnect_plugin_runtime.exceptions import PluginException
+from parameterized import parameterized
```

Comment:
You'll need to add this to the requirements.txt for the plugin (assuming it's this https://github.com/wolever/parameterized), otherwise the unit test runner will not pull down the dependency and this script will fail. 

The unit test check right now is passing because it's erroneously not finding the unit tests here 🤔

Reply:
> I updated the requirements.txt file, but the unit tests were successful because the unit_test folder still cannot be found. I checked a few other PRs that have unit tests and they have this problem too :)

Yeah a bug was introduced into the test runner a while back - it's fixed now!